components/App.js

```diff
@@ -229,6 +228,16 @@ export const onclickBindings = {
       }
     })
     setState({ menu });
+  },
+  'twoD-browse-card': e => {
+    const { menu } = getState();
+    menu.browse.selectedId = parseInt(e.name, 10);
+    menu.browse.items.forEach((item) => {
```

Comment:
I don't think `forEach` is the right style for this.

Probably the intent is `menu.browse.selectedItem = menu.browse.items.find(...)`.